Week2/homework/maartjes_work.js

```diff
@@ -1,47 +1,57 @@
 'use strict';
 
-const monday = [
-  {
-    name: 'Write a summary HTML/CSS',
-    duration: 180
-  },
-  {
-    name: 'Some web development',
-    duration: 120
-  },
-  {
-    name: 'Fix homework for class10',
-    duration: 20
-  },
-  {
-    name: 'Talk to a lot of people',
-    duration: 200
-  }
+const monday = [{
+        name: 'Write a summary HTML/CSS',
+        duration: 180
+    },
+    {
+        name: 'Some web development',
+        duration: 120
+    },
+    {
+        name: 'Fix homework for class10',
+        duration: 20
+    },
+    {
+        name: 'Talk to a lot of people',
+        duration: 200
+    }
 ];
 
-const tuesday = [
-  {
-    name: 'Keep writing summary',
-    duration: 240
-  },
-  {
-    name: 'Some more web development',
-    duration: 180
-  },
-  {
-    name: 'Staring out the window',
-    duration: 10
-  },
-  {
-    name: 'Talk to a lot of people',
-    duration: 200
-  },
-  {
-    name: 'Look at application assignments new students',
-    duration: 40
-  }
+const tuesday = [{
+        name: 'Keep writing summary',
+        duration: 240
+    },
+    {
+        name: 'Some more web development',
+        duration: 180
+    },
+    {
+        name: 'Staring out the window',
+        duration: 10
+    },
+    {
+        name: 'Talk to a lot of people',
+        duration: 200
+    },
+    {
+        name: 'Look at application assignments new students',
+        duration: 40
+    }
 ];
 
-const tasks = monday.concat(tuesday);
+const tasks = monday.concat(tuesday)
+let minutesToHours = tasks.filter(a => a.duration >= 120).map(h => h.duration / 60);
```

Comment:
You mixed up the order of first 2 steps here. While that does not matter much in this case, you should read the instructions carefully.